src/Markdig.Tests/Markdig.Tests.csproj

```diff
@@ -1,20 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
I was under the impression we had multi-targeted tests as well.
If we haven't done so before I guess keeping just 6.0 here is fine.